src/__tests__/index.js

```diff
@@ -152,6 +165,12 @@ test(
     4
 );
 
+test('should handle background value shorthand',
+    assertColourStops,
+    `header{background:resemble-image(url("${image}")) no-repeat center / cover}`,
```

Comment:
Perhaps we want to test that the gradient makes it to the end of the value? Something like this:

```js
test('should handle background value shorthand', t => {
  const fixture = `header{background:resemble-image(url("${image}")) no-repeat center / cover}`;
  return postcss(plugin()).process(fixture).then((result) => {
    t.is(result.root.first.nodes[0].value.indexOf('linear-gradient'), 30); // replace with the actual index!
    return assertColourStops(t, fixture, 4);
  });
});